CONTRIBUTING.md

```diff
@@ -17,6 +17,10 @@ Here are the guidelines we'd like you to follow:
   - [Releasing MDC-Web](#releasing-mdc-web)
 - ["What's the core team up to?"](#whats-the-core-team-up-to)
 
+## Find an Issue to Work On
+
+Material Components Web uses GitHub to file and track issues. To find an issue you'd like to work on, filter the issues list by the `help wanted` label, or [click here](https://github.com/material-components/material-components-web/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22). If you have found a bug, or would like to request a feature not represented in the list of GitHub issues, please refer to the documentation for [Contributing](https://github.com/material-components/material-components/blob/develop/CONTRIBUTING.md#issues-and-bugs)
```

Comment:
instead of having "click here", could we just make the "help wanted label" text into a link to that url?